ompi/Makefile.am

```diff
@@ -128,9 +129,9 @@ DIST_SUBDIRS = \
 
 # Build the main MPI library
 
-lib_LTLIBRARIES = libmpi.la
-libmpi_la_SOURCES =
-libmpi_la_LIBADD = \
+lib_LTLIBRARIES = lib@OMPI_LIB_PREFIX@.la
```

Comment:
Ah, you want a suffix.

Ok, in classic "Jeff wants a long name" form, I'd like `OMPI_MPI_LIBRARY_NAME`.  I'd settle for `OMPI_MPI_LIB_NAME`, too.  I think the extra `MPI` is important in there because, as a project, OMPI has *lots* of libraries.  In this case, you're specifically talking about the MPI library.

But then again, we have *several* MPI libraries...  So it would be good to see this for *all* the MPI libraries -- Fortran and C++ too.  E.g.:

* lib$(OMPI_MPI_LIB_NAME).la
* lib$(OMPI_MPI_LIB_NAME)_mpifh.la
* ...etc.